astropy/tests/helper.py

```diff
@@ -200,3 +201,68 @@ def assert_quantity_allclose(actual, desired, rtol=1.0e-7, atol=None, **kwargs):
     np.testing.assert_allclose(
         *_unquantify_allclose_arguments(actual, desired, rtol, atol), **kwargs
     )
+
+
+def assert_objects_equal(actual, desired, /, *, attrs=None, compare_class=True):
+    """Compare ndarray objects the attribute level.
```

Comment:
This should be "Compare objects by their shape and attributes, including those provided by .info"